python/tests/kernel/test_kernel_qvector_state_init.py

```diff
@@ -410,3 +410,17 @@ def kernel2():
     assert '1100' in counts
     assert '0011' in counts
     assert '0000' in counts
+
+def test_invalid_arg_error_msg():
+    cudaq.reset_target()
+
+    c = np.array([1. / np.sqrt(2.) + 0j, 0., 0., 1. / np.sqrt(2.)],
+                 dtype=complex)
```

Comment:
Is there a way to wrap this in the `State` class? Or is that already a test somewhere?

Reply:
Yes, normally users would need to wrap it in a `State` class to call this kernel signature. There are tests for this case, e.g. 

https://github.com/NVIDIA/cuda-quantum/blob/ea0f950bc31c1365708f45c6ffbdeb44d9939c67/python/tests/kernel/test_kernel_qvector_state_init.py#L75-L81